providers/embedded_ansible.md

```diff
@@ -0,0 +1,224 @@
+## Embedded Ansbile - via AWX in a docker container
+
+ManageIQ supports more than one way (`manageiq/lib/embedded_ansible*`) of connecting to the Embedded Ansible provider.
+
+Here, we'll set up `DockerEmbeddedAnsible`, introduced in [ManageIQ/manageiq#16205](https://github.com/ManageIQ/manageiq/pull/16205) - this will download an awx docker container, and set local manageiq to connect to it.
+
+
+
+### Dependencies
+
+You need docker.
+
+On a mac, you also need `docker-machine`:
+
+```
+brew install docker docker-machine
+
+# read the output and start the service
+
+# run this in your .bashrc, or every shell where needed
+eval `docker-machine env default`
+```
+
+
+### DB config
+
+Your PostgreSQL must be configured to allow connections from the docker container, so that AWX can connect to manageiq's database.
+
+Your ManageIQ DB user (the one in `manageiq/config/database.yml`) must have the `SUPERUSER` privilege (or at least needs to be able to create roles and databases).
+
+
+Config file locations:
+
+  * Debian: `/etc/postgresql/9.6/main/`
+  * Fedora: `/var/lib/pgsql/data/`
+  * MacOSX: `/usr/local/var/postgres/`
+
+
+Make sure your `postgresql.conf` contains this line:
+
+```
+listen_addresses = '*'
+```
+
+
+Make sure your `pg_hba.conf` contains:
+
+```
+host	all	all	172.17.0.1/24	md5
+```
+
+Mac users: you may also need to add this one, for `docker-machine`.
+
+```
+host all all 192.168.99.0/24 md5
+```
+
+
+Ensure your DB user (the one in `config/database.yml`) has `superuser` rights..
```

Comment:
I don't think this needs to be specified here as this is suggested in the [main developer setup document](https://github.com/ManageIQ/guides/blob/master/developer_setup.md#developer-setup)

Reply:
I'd keep it just to make sure :). It surprised me so it could surprise more people, and ... assuming every devel has removed any security from their postgres install is probably wrong, even though those instructions mention it.

Reply:
> I don't think this needs to be specified here as this is suggested in the main developer setup document
 
Well I don't have it that way. Security is one of the reasons.

This setup adds more stuff that depends on the superadmin config?

I don't think we should be doing that and I definetly agree that if it's really needed we write it here (again).

Reply:
> This setup adds more stuff that depends on the superadmin config?

Unfortunately the worker needs to create a DB and a role, so yes. :(

Agreed it's a bad idea, in the gitter discussion @Fryguy this should be something that should be optional for the more exotic setups ... but for now we seem to be stuck with it, yes.

Reply:
So, you can't `rake evm:db:reset` either?

Reply:
If you run differently then the default guides, you're going to have issues because we can't know how you diverged from the guides.  Please either change the default guides to be "right", include sensible alternatives in the guides, or use the defaults.  It's hard enough to maintain these guides when people use the defaults, if they don't use the defaults, the guides become stale and it's easy to accidentally break someone else's setup.